packages/shared/src/components/LoginButtons.tsx

```diff
@@ -8,7 +8,13 @@ import AuthContext from '../contexts/AuthContext';
 import { apiUrl } from '../lib/config';
 import AnalyticsContext from '../contexts/AnalyticsContext';
 
-export default function LoginButtons(): ReactElement {
+export interface LoginButtonsProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
Some small notes, not breaking, but the `export` is not needed here as we are not using it outside this file.

Also the `extends HTMLAttributes<HTMLDivElement>` doesn't add anything here as we are not passing more props.

If you are remove those we should be ready to go 🎉